test/extended/util/image_helpers.go

```diff
@@ -74,3 +74,9 @@ func StartBuild(jsonFile, buildPrefix string, oc *CLI) {
 	_, berr := oc.Run("start-build").Args(buildPrefix).Output()
 	o.Expect(berr).NotTo(o.HaveOccurred())
 }
+
+//CreateResource creates the resources from the supplied json file (not a template); ginkgo error checking included
+func CreateResource(jsonFile string, oc *CLI) {
```

Comment:
might be more generically useful if it returned the error rather than ginkgo'ing it.  someone might have a test case where they want to create and do expect an error.